docs/docs/decorators/commands/simplecommandoption.md

```diff
@@ -37,6 +37,12 @@ To declare an option you simply use the `@SimpleCommandOption` decorator before
   }
 ```
 
+## User instance inside DM only
+
+You will receive `ClientUser` if the user mentions the bot in DM. If the user mentions himself/herself, You will receive a `User`. Otherwise `null`.
```

Comment:
```suggestion
You will receive `ClientUser` if the user mentions the bot in DM. If the user mentions himself/herself, You will receive a `User`. Otherwise, you'll receive an error.
```